### PR TITLE
update guide on Markdown tables

### PR DESCRIPTION
**Pull request by abhiditi**
- Please format your code first: `./gradlew spotlessApply`
- Please describe *what* you're trying to achieve and *why*
- Please sign each commit `git commit -s ...`

### CODE REVIEW

File: docs/guide.md [hunk 1]
```diff
@@ -111,6 +111,9 @@ types can be added in the form `[package.Interface.method]`. For properties the
 explicitly. Without suffix the reference will link to the getter in languages which have
 getters and setters.
 
+**Note:** Although, gluecodium supports Markdown tables, but the rendered view of the tables are 
+different from normal Markdown tables. Hence, adding Markdown tables is not recommend.
```
Comment (DanielKamkha) [inline docs/guide.md:115]: This is simply not true. Gluecodium renders MD tables from its IDL files into JavaDoc HTML tables. For the rest of the output languages, the MD tables are preserved *verbatim*. Therefore, the statement about the tables being "different" is false.

**Review by DanielKamkha — CHANGES_REQUESTED** [review]

File: docs/guide.md (hunk 1 above)
Comment (abhiditi) [reply docs/guide.md:115]: where can we find and edit this JavaDoc file? is it possible?

Comment (DanielKamkha) [reply docs/guide.md:115]: LIME IDL doc comments support Markdown (MD). Gluecodium renders this Markdown into markup syntax appropriate to each output language. For Java output the markup language is HTML (as per JavaDoc specs). For C++, Swift, and Dart, the markup language is the same MD.

There is no "JavaDoc file" to edit. If you have an issue with how an MD table is rendered into HTML, please open a bug ticket or a feature request, depending on whether the table is broken or just looks not nice enough.

If you have an issue with how an MD table is rendered into MD, Gluecodium has nothing to do with that. It preserves the table verbatim (for C++, Swift, and Dart).

Comment (abhiditi) [reply docs/guide.md:115]: Thanks @DanielKamkha , I have an issue with how an MD table is rendered into HTML and I have created a bug ticket for it.

**Pull request closed without merging**